### DISCUSSION TITLE
USB Micro Input Current

### DISCUSSION THREAD

**Discussion by marvin-steinke** [PiSupply/PiJuice#850] [Q&A]
I plan to use the 22w pijuice solar module to power the hat via its micro usb input. In the [status cli menu](https://github.com/PiSupply/PiJuice/blob/master/Software/README.md#status) the state of the usb micro power input is displayed as either NOT_PRESENT, BAD, WEAK or PRESENT. Is it possible to get the state in Volts and Amperes?

**Comment by quitmeyer** [comment]
would  also like to see volts and amps
